clusterloader2/run-e2e.sh

```diff
@@ -22,5 +22,10 @@ CLUSTERLOADER_ROOT=$(dirname "${BASH_SOURCE}")
 export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
 export KUBEMARK_ROOT_KUBECONFIG="${KUBEMARK_ROOT_KUBECONFIG:-${HOME}/.kube/config}"
 
+# Deploy a secret that contains the e2e Google credentials
+if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
```

Comment:
1. Do we have access to this env-var here? [I hope we do, just asking if you verified it.]

2. Please add :- to avoid using unset var (both here and below)

Reply:
The env var is set by prow afaik and is inherited for all prow jobs.